src/classes/CMT_MetadataAPI.cls

```diff
@@ -159,11 +180,17 @@ public class CMT_MetadataAPI {
             crlpSettings.CMT_API_Status__c = JSON.serializePretty(statusMap).left(255);
             // THIS WILL NEED TO BE REFACTORED IF THIS CMT DEPLOYMENT CODE IS USED BY SOMETHING OTHER THAN ROLLUPS
             crlpSettings.Customizable_Rollups_Enabled__c = (isSuccess || crlpSettings.Customizable_Rollups_Enabled__c == true);
-            if (!Test.isRunningTest()) {
-                upsert crlpSettings;
-            }
+
```

Comment:
I didn't touch this code above but as it says if we ever use the deployment for stuff other than CMT we'd probably want to refactor it